Validation/RecoMuon/python/PostProcessorHLT_cff.py

```diff
@@ -4,25 +4,32 @@
 from DQMServices.Core.DQMEDHarvester import DQMEDHarvester
 
 from Validation.RecoMuon.PostProcessor_cff import postProcessorMuonTrack
+
 postProcessorMuonTrackHLT = postProcessorMuonTrack.clone(
-    subDirs = ["HLT/Muon/MuonTrack/*"]
+    subDirs=["HLT/Muon/MuonTrack/*"]
```

Comment:
why are the spaces around `=` signs removed? There's no universal consensus on the python formatting (see https://github.com/cms-sw/cmssw/issues/45706) but I think this option is clearly disfavored.